.changeset/quiet-suns-eat.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@CiaranMn, I've added a changeset, do I need to do a follow-up PR after release to change local dependency versions on `block-scripts` to the updated version? Or does the changeset release thing handle that

Reply:
changesets does that